### ISSUE TITLE
Errors on CentOS

### ISSUE THREAD

**Issue by rdzar** [mariano/node-db-mysql#6]
While instelling with npm I run into the following errors

```
[10/11] cxx: src/mysql_bindings.cc -> build/default/src/mysql_bindings_1.o
../src/connection.cc: In member function ‘virtual void node_db_mysql::Connection::open()’:
../src/connection.cc:70: error: cannot convert ‘uint32_t*’ to ‘const char*’ for argument ‘3’ to ‘int mysql_options(MYSQL*, mysql_option, const char*)’
../src/connection.cc:74: error: cannot convert ‘bool*’ to ‘const char*’ for argument ‘3’ to ‘int mysql_options(MYSQL*, mysql_option, const char*)’
../src/connection.cc:77: error: cannot convert ‘bool*’ to ‘const char*’ for argument ‘3’ to ‘int mysql_options(MYSQL*, mysql_option, const char*)’
../src/connection.cc:80: error: cannot convert ‘uint32_t*’ to ‘const char*’ for argument ‘3’ to ‘int mysql_options(MYSQL*, mysql_option, const char*)’
../src/connection.cc:84: error: cannot convert ‘uint32_t*’ to ‘const char*’ for argument ‘3’ to ‘int mysql_options(MYSQL*, mysql_option, const char*)’
../src/result.cc: In member function ‘virtual node_db_mysql::Result::Column* node_db_mysql::Result::column(uint16_t) const’:
../src/result.cc:150: warning: comparison is always false due to limited range of data type
../lib/node-db/query.cc: In member function ‘std::string node_db::Query::value(v8::Local<v8::Value>, bool, bool) const’:
../lib/node-db/query.cc:1373: warning: passing ‘double’ for argument 1 to ‘std::string node_db::Query::fromDate(uint64_t) const’
```

I thought there was something with the headers, checked included dir, but that seems fine to me.

```
-I/usr/include/mysql
```

Paths seems fine to be too.

```
Checking for program mysql_config        : /usr/bin/mysql_config 
Checking for library mysqlclient_r       : yes 
```

Tried with `EXPORT` setting it to `/usr/lib64/mysql/mysql_config`, but that did not do the trick either.
This might be something stupid on my hand, but I'm not seeing it right now :)

**Comment by mariano** [comment]
It doesn't seem to be your fault, but differences in the systems on which I've been compling, and CentOS.

I'll look it over today and fix the issues.

**Comment by rdzar** [comment]
Cool, thanks!

**Comment by mariano** [comment]
@renedx: This issue should now be resolved in the latest release of db-mysql. So try installing it again, and let me know how it goes.

**Comment by rdzar** [comment]
Thats fast, thank you. It works now!

IF you find the 3 warnings interesting, but it works :)

```
../src/result.cc: In member function ‘virtual node_db_mysql::Result::Column* node_db_mysql::Result::column(uint16_t) const’:
../src/result.cc:150: warning: comparison is always false due to limited range of data type
../lib/node-db/query.cc: In member function ‘std::string node_db::Query::fromDate(double) const’:
../lib/node-db/query.cc:1429: warning: converting to ‘time_t’ from ‘double’
```

**Comment by mariano** [comment]
@renedx: thanks! Warnings should be fixed in 0.5.9 which was just released
